packages/core/src/Stores/client-store.js

```diff
@@ -1243,7 +1243,11 @@ export default class ClientStore extends BaseStore {
             }
             if (redirect_url) {
                 const redirect_route = routes[redirect_url].length > 1 ? routes[redirect_url] : '';
-                if (search_params?.get('action') === 'reset_password') {
+                if (
+                    ['payment_agent_withdraw', 'payment_withdraw', 'reset_password'].includes(
+                        search_params?.get('action')
```

Comment:
Let's assign this logical part inside a well named variable, later use it inside the if statement. That way, it'll be more readable.

`['payment_agent_withdraw', 'payment_withdraw', 'reset_password'].includes(
                        search_params?.get('action')
                    )`